view/frontend/templates/js/replacejs.phtml

```diff
@@ -1091,10 +1101,9 @@ ob_start();
         ///////////////////////////////////////////////////////////
         }
 
-        // When item added to cart, refresh order and mark data modified so always present button knows new item added
-        // to cart and can animate cart
+        // When item added to cart, set flag to animate always present button when we catch cart updating
         $(document).on("ajax:addToCart", function () {
```

Comment:
Can we gate listener behind `boltConfig.always_present_checkout`? I should have done this to begin with but didn't.

Reply:
Good idea! Done.